MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -471,62 +471,31 @@ public bool IsTextSubtitleStream
 
         private string GetResolutionText()
         {
-            var i = this;
-
-            if (i.Width.HasValue && i.Height.HasValue)
+            if (!this.Width.HasValue || !this.Height.HasValue)
             {
-                var width = i.Width.Value;
-                var height = i.Height.Value;
-
-                if (width >= 3800 || height >= 2000)
-                {
-                    return "4K";
-                }
-
-                if (width >= 2500)
-                {
-                    if (i.IsInterlaced)
-                    {
-                        return "1440i";
-                    }
-
-                    return "1440p";
-                }
-
-                if (width >= 1900 || height >= 1000)
-                {
-                    if (i.IsInterlaced)
-                    {
-                        return "1080i";
-                    }
-
-                    return "1080p";
-                }
-
-                if (width >= 1260 || height >= 700)
-                {
-                    if (i.IsInterlaced)
-                    {
-                        return "720i";
-                    }
-
-                    return "720p";
-                }
-
-                if (width >= 700 || height >= 440)
-                {
-                    if (i.IsInterlaced)
-                    {
-                        return "480i";
-                    }
-
-                    return "480p";
-                }
-
-                return "SD";
+                return null;
             }
 
-            return null;
+            var width = this.Width.Value;
```

Comment:
Can I ask why the need to prefixed everything with _this_?

Also, as the Width and Height have been null checked at this point, why not just use Width and Height instead of assigning to new variables?

Reply:
It's mainly to mimic what was there before, since it used `var i = this;`.

The variables are there for readability, mainly.

Reply:
personally, i'd loose the 'this' as it's implied - and doesn't seem to be the style elsewhere.